federation/cluster/federation-up.sh

```diff
@@ -82,7 +82,8 @@ function init() {
       "${FEDERATION_NAME}" \
       --host-cluster-context="${HOST_CLUSTER_CONTEXT}" \
       --dns-zone-name="${DNS_ZONE_NAME}" \
-      --image="${kube_registry}/hyperkube-amd64:${kube_version}"
+      --image="${kube_registry}/hyperkube-amd64:${kube_version}" \
+      --apiserver-arg-overrides="--storage-backend=etcd2"
```

Comment:
I am not sure, if enough analysis or update (if there is any that is needed) is done in the federation API server code to move to etcd3.
Given the previous, trying to be cautious not to break something.
as far as I know @madhusudancs and @shashidharatd found some issues with using etcd3 backend; probably they can comment better.